scarb/tests/e2e/build.rs

```diff
@@ -193,14 +193,16 @@ fn compile_with_invalid_empty_name() {
         .current_dir(&t)
         .assert()
         .code(1)
-        .stdout_matches(
-            "\
-error: failed to parse manifest at `[..]/Scarb.toml`
-
-Caused by:
-    empty string cannot be used as package name for key `package.name`
-",
-        );
+        .stdout_matches(indoc! {"\
+            error: failed to parse manifest at `[..]/Scarb.toml`
+
+            Caused by:
+                TOML parse error at line 3, column 20
+                  |
+                3 |             name = \"\"
+                  |                    ^^
+                empty string cannot be used as package name
+                \n",});
```

Comment:
```suggestion
        .stdout_matches(indoc! {r#"
            error: failed to parse manifest at `[..]/Scarb.toml`

            Caused by:
                TOML parse error at line 3, column 20
                  |
                3 |             name = ""
                  |                    ^^
                empty string cannot be used as package name
        "#});
```

Apply this everywhere here. See how other tests do this: https://github.com/software-mansion/scarb/blob/dc08364119a5d19a132cddd566d9025d12e206d6/scarb/tests/e2e/build.rs#L65-L79

Reply:
Resolving after discussing offline.